src/qt/walletmodel.cpp

```diff
@@ -514,6 +514,13 @@ bool WalletModel::bumpFee(uint256 hash, uint256& new_hash)
     questionString.append("</td><td>");
     questionString.append(BitcoinUnits::formatHtmlWithUnit(getOptionsModel()->getDisplayUnit(), new_fee));
     questionString.append("</td></tr></table>");
+
+    //Check if "Coin Control Features" options is enabled to display warning in "Confirm fee bump" Window
```

Comment:
nit, not sure this comment is needed, but if you keep it
```suggestion
    // Display warning in the "Confirm fee bump" window if the "Coin Control Features" option is enabled
```